lib/tasks/staging_seeds.rake

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+namespace :comfy do
+  
+  # Locally stored seeds - assumes you already have the local folder - 
+  # it won't create it 
+  LOCAL = (ComfortableMexicanSofa.config.seeds_path + '/protected-planet').freeze
+  REMOTE = 'ProtectedPlanet/current/db/cms_seeds/protected-planet'.freeze
+  PP_STAGING = 'new-web.pp-staging.linode.protectedplanet.net'.freeze
+  PP_PRODUCTION = 'new-web.pp-production.linode.protectedplanet.net'.freeze
```

Comment:
We are not really using this at the moment as we are mainly testing on the local environment.
Perhaps you could make a condition checking the `Rails.env` and using the local address or PP_PRODUCTION based on that?

Reply:
Hmm, how about I could use a dynamic variable for the remote folder like so: 
`'new-web.pp-#{Rails.env}.linode.protectedplanet.net'`

Reply:
I'm not sure I fully understand here.
So this taks should be used also so that production can get the seeds from staging and import those when ready.
So both when running this task in development and production shoul use the staging server as the main reference.

Reply:
Ah I see, I was always under the impression that we'd want to obtain the content from production because that's the one that gets updated most frequently and has the right content by CLS...or is it not the case with Protected Planet?

Reply:
The main reason for this work I'm aware of, is that the CMS will get first populated on staging so not to disrupt production and so that we could do all the necessary checks there before deploying live. This mean that once everything is ready on staging with regards the CMS, we would need to run this task on production to fetch the CMS seeds from staging and also run the import task. Hope it makes sense